packages/@okta/vuepress-site/docs/guides/email-customization/customize-email-templates/index.md

```diff
@@ -57,25 +57,45 @@ The new templating syntax is different from the EL expression-based Okta email t
+#### Access to all User Profile attributes
 
-`${f:formatTimeDiffHoursNowInUserLocale(org.activationTokenExpirationHours)}`
+You can reference any Okta User Profile attribute in your email templates. The reference notation is `$user.profile.attributeName`, where `attributeName` is an attribute from the Okta User Profile. For example, use `$user.profile.displayName` to reference the User Profile `displayName` attribute.
 
-You would call functions in the Velocity Templating syntax like this:
+See [Profile object](/docs/reference/api/users/#profile-object) for more information on the available User Profile attributes.
 
-`$f.formatTimeDiffHoursNowInUserLocale($org.activationTokenExpirationHours)`
--->
+#### Functions for email templates
 
-#### Access to all User Profile attributes
+In addition to customizing your emails with variables, you can use the following functions in each of the email templates. Functions are useful to normalize the dynamic output of variables, such as lowercasing a string, or producing a localized date for the email's recipient user. 
+
+Variables used for function parameters must match the function data type. For example, you can't use a string variable with the `formatTimeDiffHourNow()` function because the parameter data must be an integer.
+
+There are slight differences between EL templating syntax and VTL syntax. For example, the following EL and VTL syntax call the same function.
 
-You can now reference any attribute in a user's Okta User Profile in your email templates. The notation is `$user.profile.attributeName`. For example, to reference the displayName attribute of the user's Okta user profile, you would use `$user.profile.displayName`.
+EL templating syntax:
+
+`${f:formatTimeDiffHoursNowInUserLocale(org.activationTokenExpirationHours)}`
+
+VTL syntax:
+
+`$f.formatTimeDiffHoursNowInUserLocale($org.activationTokenExpirationHours)`
 
-See [Profile object](/docs/reference/api/users/#profile-object) for more information on the available user profile attributes.
+| Expression                                                    | Definition                                                                                                          |
+|---------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| replace(String input, String matchString, String replacement) | Replaces all occurrences of the match string with the replacement string                                           |
+| substringAfter(String input, String matchString)              | Returns the input substring after the occurrence of a given match string, or an empty string if there isn't |
+| substringBefore(String input, String matchString)             | Returns the input substring before the occurrence of a given match string, or an empty string if no occurrence of match string exists in input string |
```

Comment:
Nit - can we change the last part of the definition to: `if no occurrence of THE match string exists in THE input string`?